ironfish/src/rpc/routes/wallet/createTransaction.ts

```diff
@@ -218,6 +220,7 @@ router.register<typeof CreateTransactionRequestSchema, CreateTransactionResponse
           expirationDelta:
             data.expirationDelta ?? node.config.get('transactionExpirationDelta'),
           expiration: data.expiration,
+          confirmations: data.confirmations,
```

Comment:
would be nice to include a unit test in createTransaction.test.ts, like what you did for sendTransaction.test.ts

Reply:
added